routes/web.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Support\Facades\Route;
+use App\Http\Controllers\ArticleController;
+use App\Http\Controllers\SessionController;
+use App\Http\Controllers\ManageController;
+use App\Http\Controllers\ManageTagController;
+use App\Http\Controllers\ManageCategoryController;
+use App\Http\Controllers\ManageArticleController;
+
+/*
+|--------------------------------------------------------------------------
+| Web Routes
+|--------------------------------------------------------------------------
+|
+| Here is where you can register web routes for your application. These
+| routes are loaded by the RouteServiceProvider within a group which
+| contains the "web" middleware group. Now create something great!
+|
+*/
+
+Route::get('/', [ArticleController::Class, 'index'])->name('article.index');
+Route::get('/article/{article}', [ArticleController::Class, 'show'])->name('article.show');
+
+Route::get('/login', [SessionController::class, 'create'])->name('session.create')->middleware('guest');
+Route::post('/login', [SessionController::class, 'store'])->name('session.store')->middleware('guest');
+Route::delete('/logout', [SessionController::class, 'destroy'])->name('session.destroy')->middleware('auth');
+
+Route::prefix('manage')->middleware(['auth', 'can:admin'])->group(function () {
```

Comment:
Some routes here has name prefix of manage. some don't. Either use it or not. Add name prefix to whole group here. Plus all manage controllers should go into different folder then it would be cleaner. Imagine if you had 10+ controllers for front and 10+ for manage. And I personally don't like that manage naming, usually it's admin. But naming is personal preference.

Reply:
The different folder and naming conventions for all Manage/Admin controllers makes a lot of sense.
I added manage to articles because it was conflicting with the ArticleController routes, but naming the group is way better.